# translation of ru.po to Russian
# translation of firstboot.master.ru.po to
# translation of ru.po to
#
# Alexandr Kanevskiy <kad@blackcatlinux.com>, 2003.
# Leonid Kanter <leon@asplinux.ru>, 2004.
# Andrew Martynov <andrewm@inventa.ru>, 2004, 2005, 2006, 2008.
# Yulia Poyarkova <ypoyarko@redhat.com>, 2006, 2010, 2011.
# Nikolay Sivov <bunglehead@gmail.com>, 2007.
# Copyright (C) 2003
msgid ""
msgstr ""
"Project-Id-Version: ru\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-05-05 09:32+0200\n"
"PO-Revision-Date: 2011-05-11 10:21\n"
"Last-Translator: Yulia <ypoyarko@redhat.com>\n"
"Language-Team: Russian\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms:  nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: KBabel 1.11.4\n"

#: ../progs/firstboot:140
msgid "You must be root to run firstboot."
msgstr "Для запуска firstboot необходимы права root."

#: ../progs/firstboot:194 ../progs/firstboot:195
msgid "No firstboot modules were found."
msgstr "Не найдены модули firstboot."

#: ../progs/firstboot:205 ../progs/firstboot:206
msgid "Could not create any firstboot interface."
msgstr "Не удалось создать ни один из интерфейсов firstboot."

#: ../firstboot/moduleset.py:92
#, python-format
msgid "Module %s did not set up its UI; removing."
msgstr "Модуль %s не настроил свой пользовательский интерфейс и будет удален."

#: ../firstboot/interface.py:79
msgid "Attempted to go back, but history is empty."
msgstr "При попытке вернуться назад обнаружено, что история возврата пуста."

#. If we were previously on the last page, we need to set the Next
#. button's label back to normal.
#: ../firstboot/interface.py:87 ../firstboot/interface.py:163
msgid "_Finish"
msgstr "_Готово"

#: ../firstboot/interface.py:184
msgid "The system must now reboot for some of your selections to take effect."
msgstr "Чтобы установки вступили в силу, необходим перезапуск системы."

#: ../firstboot/interface.py:243
msgid "_Back"
msgstr "_Назад"

#: ../firstboot/interface.py:250
msgid "_Forward"
msgstr "_Вперёд"

#: ../firstboot/interface.py:277
#, python-format
msgid "Module %s did not set up its UI, removing."
msgstr "Модуль %s не настроил свой пользовательский интерфейс и будет удален."

#: ../firstboot/interface.py:347 ../firstboot/interface.py:348
msgid "moveToPage must be given a module title or page number."
msgstr "moveToPage: необходимо предоставить название модуля или номер страницы. "

#: ../firstboot/interface.py:432
msgid "Unable to create the screenshot dir; skipping."
msgstr "Не удалось создать каталог снимков экрана, пропускаем."

#: ../firstboot/interface.py:472 ../firstboot/interface.py:473
#, python-format
msgid "No module exists with the title %s."
msgstr "Нет модуля с заголовком %s."

#: ../firstboot/loader.py:87
#, python-format
msgid "Skipping old module %s that has not been updated."
msgstr "Пропускается старый модуль %s, который не был обновлен."

#: ../firstboot/loader.py:89 ../firstboot/loader.py:93
#, python-format
msgid ""
"Error loading module %(module)s:\n"
"%(error)s"
msgstr ""
"Ошибка при загрузке модуля %(module)s:\n"
"%(error)s"

#: ../firstboot/loader.py:103
#, python-format
msgid "Module %s does not contain a class named moduleClass; skipping."
msgstr "Модуль %s не содержит класса moduleClass; пропускаем."

#: ../firstboot/loader.py:114
#, python-format
msgid "Module %(module)s does not contain the required attribute %(attr)s; skipping."
msgstr "Модуль %(module)s не содержит необходимый атрибут %(attr)s. Пропускается."

#: ../modules/date.py:39 ../modules/date.py:40
msgid "Date and Time"
msgstr "Дата и время"

#: ../modules/date.py:61
msgid "Please set the date and time for the system."
msgstr "Пожалуйста, установите дату и время системы."

#: ../modules/rootpassword.py:47 ../modules/rootpassword.py:48
msgid "Root Password"
msgstr "Пароль root"

#: ../modules/rootpassword.py:62
msgid "The passwords do not match. Please enter the password again."
msgstr "Пароли не совпадают. Повторите ввод."

#: ../modules/rootpassword.py:78
msgid ""
"You can change the root password by filling \n"
"the following fields. If you leave them empty the password will not be "
"changed."
msgstr ""
"Пароль root можно изменить ниже.\n"
"Чтобы не изменять пароль, оставьте поля пустыми."

#: ../modules/rootpassword.py:90
msgid "New Root Password:"
msgstr "Новый пароль root:"

#: ../modules/rootpassword.py:94
msgid "Confirm Root Password:"
msgstr "Подтверждение пароля root:"

#: ../modules/create_user.py:42 ../modules/create_user.py:43
msgid "Create User"
msgstr "Пользователь"

#: ../modules/create_user.py:95
msgid ""
"You did not set up an user account capable of logging into the system.\n"
"Are you sure you want to continue?"
msgstr ""
"Не выбрана учётная запись для авторизации в системе.\n"
"Вы действительно хотите продолжить?"

#: ../modules/create_user.py:116
msgid "You must enter and confirm a password for this user."
msgstr "Требуется ввести и подтвердить пароль пользователя."

#: ../modules/create_user.py:123
msgid "The passwords do not match.  Please enter the password again."
msgstr "Пароли не совпадают.  Введите пароль снова."

#: ../modules/create_user.py:135
#, python-format
msgid ""
"The username '%s' is a reserved system account.  Please specify another "
"username."
msgstr ""
"Имя %s &mdash; зарезервированная системная учётная запись. Укажите другое "
"имя."

#: ../modules/create_user.py:154
#, python-format
msgid ""
"A home directory for user %s already exists. Would you like to continue, "
"making the new user the owner of this directory and all its contents?  Doing "
"so may take a while to reset permissions and any SELinux labels.  Would you "
"like to reuse this home directory?  If not, please choose a different "
"username."
msgstr ""
"Домашний каталог пользователя %s уже существует. Вы действительно хотите "
"сделать нового пользователя владельцем этого каталога и его содержимого?  В "
"этом случае переназначение разрешений и меток SELinux может занять некоторое "
"время. Использовать этот домашний каталог повторно? Если нет, выберите "
"другое имя пользователя."

#: ../modules/create_user.py:197
#, python-format
msgid "Fixing attributes on the home directory for %s.  This may take a few minutes."
msgstr ""
"Фиксируются атрибуты домашнего каталога для %s. Это может занять несколько "
"минут."

#: ../modules/create_user.py:217
#, python-format
msgid ""
"Problems were encountered fixing the attributes on some files in the home "
"directory for %(user)s.  Please refer to %(path)s for which files caused the "
"errors."
msgstr ""
"Не удалось присвоить атрибуты некоторым файлам в домашнем каталоге "
"%(user)s. Обратитесь к %(path)s для получения информации о вызвавших ошибки "
"файлах."

#: ../modules/create_user.py:237
msgid ""
"You must create a 'username' for regular (non-administrative) use of your "
"system.  To create a system 'username', please provide the information "
"requested below."
msgstr ""
"Требуется создать пользователя для повседневного (не административного) "
"использования системы. Для этого введите необходимые данные."

#: ../modules/create_user.py:257
msgid "_Username:"
msgstr "_Имя пользователя:"

#: ../modules/create_user.py:264
msgid "Full Nam_e:"
msgstr "Полное им_я:"

#: ../modules/create_user.py:271
msgid "_Password:"
msgstr "_Пароль:"

#: ../modules/create_user.py:278
msgid "Confir_m Password:"
msgstr "По_дтвердите пароль:"

#: ../modules/create_user.py:287
msgid ""
"If you need to use network authentication, such as Kerberos or NIS, please "
"click the Use Network Login button."
msgstr ""
"Если требуется использовать проверку подлинности по сети, например Kerberos "
"или NIS, нажмите кнопку «Сетевая аутентификация»."

#: ../modules/create_user.py:296
msgid "Use Network _Login..."
msgstr "Сетевая аут_ентификация..."

#: ../modules/create_user.py:304
msgid ""
"If you need more control when creating the user (specifying home directory, "
"and/or UID), please click the Advanced button."
msgstr "Для настройки других параметров (домашнего каталога, UID) нажмите кнопку «Дополнительно»."

#: ../modules/create_user.py:314
msgid "_Advanced..."
msgstr "_Дополнительно..."

#: ../modules/create_user.py:352
msgid "Please wait"
msgstr "Подождите"

#: ../modules/additional_cds.py:43 ../modules/additional_cds.py:44
msgid "Additional CDs"
msgstr "Дополнительные CD"

#: ../modules/additional_cds.py:55
msgid ""
"Please insert the disc labeled \"Red Hat Enterprise Linux Extras\" to allow "
"for installation of third-party plug-ins and applications.  You may also "
"insert the Documentation disc, or other Red Hat-provided discs to install "
"additional software at this time."
msgstr ""
"Вставьте диск с надписью «Red Hat Enterprise Linux Extras» для установки "
"приложений и дополнительных модулей третьих фирм.  Вы также можете вставить "
"диск с документацией или любой другой диск, предоставленный компанией Red "
"Hat, для установки дополнительного программного обеспечения на данном этапе."

#: ../modules/additional_cds.py:61
msgid "Please insert any additional software install cds at this time."
msgstr "Вставьте диск с дополнительным программным обеспечением."

#: ../modules/additional_cds.py:65
msgid ""
"\n"
"\n"
"To enable runtime support of 32-bit applications on the Intel Itanium2 "
"architecture you must install the Intel Execution Layer package from the "
"Extras disc now."
msgstr ""
"\n"
"\n"
"Для выполнения 32-битных приложений на архитектуре Intel Itanium2 необходимо "
"дополнительно установить Intel Execution Layer с диска с дополнительным "
"программным обеспечением (Extras)."

#: ../modules/additional_cds.py:73
msgid "Install..."
msgstr "Установка..."

#: ../modules/additional_cds.py:110
msgid ""
"A CD-ROM has not been detected.  Please insert a CD-ROM in the drive and "
"click \"OK\" to continue."
msgstr "CD-ROM найден.  Вставьте CD в привод и нажмите «OK» для продолжения."

#: ../modules/additional_cds.py:146
msgid "The autorun program cannot be found on the CD. Click \"OK\" to continue."
msgstr "Программа autorun не найдена на CD. Нажмите «OK» для продолжения."

#: ../modules/eula.py:37 ../modules/eula.py:38
msgid "License Information"
msgstr "Информация о лицензии"

#: ../modules/eula.py:46
msgid ""
"Do you want to reread or reconsider the Licence Agreement?  If not, please "
"shut down the computer and remove this product from your system. "
msgstr ""
"Хотели бы перечитать лицензию или ещё подумать над условиями соглашения? "
"Если нет, выключите компьютер и удалите этот программный продукт."

#: ../modules/eula.py:53
msgid "_Reread license"
msgstr "_Перечитать"

#: ../modules/eula.py:54
msgid "_Shut down"
msgstr "_Выключить"

#: ../modules/eula.py:95
msgid "_Yes, I agree to the License Agreement"
msgstr "_Да, я принимаю условия соглашения"

#: ../modules/eula.py:96
msgid "N_o, I do not agree"
msgstr "_Нет, я не согласен"

#: ../modules/welcome.py:35 ../modules/welcome.py:36
msgid "Welcome"
msgstr "Добро пожаловать"

#: ../modules/welcome.py:45
msgid ""
"There are a few more steps to take before your system is ready to use.  The "
"Setup Agent will now guide you through some basic configuration.  Please "
"click the \"Forward\" button in the lower right corner to continue"
msgstr ""
"Осталось ещё несколько шагов, после чего ваша система будет готова к работе. "
"В этом вам поможет помощник по настройке. Нажмите кнопку продолжения."

